src/properties.go

```diff
@@ -17,7 +17,11 @@ const (
 	Postfix Property = "postfix"
 	// ColorBackground color the background or foreground when a specific color is set
 	ColorBackground Property = "color_background"
-	// IgnoreFolders folders to ignore and not run the segment logic
+	// IncludeFolders folders to be included for the segment logic
+	IncludeFolders Property = "include_folders"
+	// ExcludeFolders folders to be excluded for the segment logic
+	ExcludeFolders Property = "exclude_folders"
+	// IgnoreFolders duplicate of ExcludeFolders
```

Comment:
that's for backwards compatibility, right? Can we mark this property as deprecated in the schema?

Reply:
This seems to be a duplicate of your comment on schema.json.  Let me know if this is asking for something different.

Reply:
Duplicate 😃